docs/versioned_docs/version-v0.4.0/01-index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Arcus Background Jobs"
+layout: default
+permalink: /
+slug: /
+sidebar_label: Welcome
+---
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.BackgroundJobs.CloudEvents?includePreReleases=true)](https://www.nuget.org/packages/Arcus.BackgroundJobs.CloudEvents/)
+
+# Installation
+
+The Arcus BackgroundJobs can be installed via NuGet:
+
+```shell
+PM > Install-Package Arcus.BackgroundJobs.CloudEvents
+```
+
+For more granular packages we recommend reading the documentation.
```

Comment:
```suggestion
For more in depth documentation we recommend reading the documentation.
```
I'd also put a link to the documentation ?

Reply:
Hmm, but here we're already at the documentation site 😅 . Maybe we could add `:` at the end as to link to the `# Features` title below.

Reply:
Removed this sentence and NuGet badge in https://github.com/arcus-azure/arcus.backgroundjobs/pull/143/commits/a611c6a3123ab76aeaeaa817b9edce9df8fb7758